### ISSUE TITLE
Error: Files unavailable

### ISSUE THREAD

**Issue by DnDnDy** [elishacloud/Silent-Hill-2-Enhancements#590]
I have previously installed SH2 and SH2EE, and previously deleted the files.

I just tried to install SH2EE again, and get this error:

> ---------------------------
>  Silent Hill 2: Enhanced Edition Setup Tool
> ---------------------------
> Error: Files unavailable
> 
> Failed to query for one or more components.
> 
> The installation cannot continue. Please try again, and if the issue persists, report it to the developers.
> 
![sh2ee error](https://user-images.githubusercontent.com/108682675/187473399-9da863aa-978e-40b7-b479-b836f0be3fe5.png)

I assume what is happening is that SH2EEsetup.exe is checking a registry key and finding it was previously installed, but then when it can't find the files, it errors instead of simply acting as if it isn't installed.

When the installer "can't find the files", that's probably why I'm running the installer.

**Comment by Polymega** [comment]
No worries. For anyone else who finds this ticket: Please redownloaded the tool from the project site. More information in this ticket: https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/588

**Comment by nipkownix** [comment]
@Polymega Should probably bump the installer version, so everyone gets this update, right?

**Comment by Polymega** [comment]
I believe you'd need to bump it within the tool as well, correct? Currently the tool is still using the same 1.0.5 version.

**Comment by nipkownix** [comment]
Yeah. I didn't bump it before because I thought we were only testing :p
Will send you an update soon.
